blockscout-ens/bens-server/src/settings.rs

```diff
@@ -28,10 +28,49 @@ impl ConfigSettings for Settings {
     const SERVICE_NAME: &'static str = "BENS";
 }
 
+// TODO: move database settings to blockscout-service-launcher
 #[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
 #[serde(deny_unknown_fields)]
 pub struct DatabaseSettings {
-    pub url: String,
+    pub connect: DatabaseConnectSettings,
+}
+
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields, rename_all = "lowercase")]
```

Comment:
Nit: Just interested, why it should be a lowercase? Does it affect anything at all?

Reply:
it fails without lowercase because we use `URL` in env variable name

```
failed to read config: enum DatabaseConnectSettings does not have variant constructor url
```